src/release-notify-action.js

```diff
@@ -1,20 +1,19 @@
 'use strict'
 
-const { logInfo, logWarning } = require('./log')
+const { logInfo } = require('./log')
 const { getLatestRelease, getUnreleasedCommits } = require('./release')
 const {
   createOrUpdateIssue,
   getLastOpenPendingIssue,
   closeIssue,
+  isSnoozed,
 } = require('./issue')
 
-async function runAction(token, staleDays, commitMessageLines) {
-  let latestRelease
+async function runAction(token, staleDate, commitMessageLines) {
+  const latestRelease = await getLatestRelease(token)
 
-  try {
-    latestRelease = await getLatestRelease(token)
-  } catch (err) {
-    return logWarning('Could not find latest release')
+  if (!latestRelease) {
+    return
```

Comment:
we should log a warning here, otherwise we don't know why the action is doing nothing (and remove it from the function being called here, as commented elsewhere)